tests/microgrid/mock_api.py

```diff
@@ -89,6 +92,7 @@ def add_component(
         self,
         component_id: int,
         component_category: ComponentCategory.V,
+        max_current: Optional[Current] = None,
```

Comment:
```suggestion
        max_current: Current | None = None,
```

Reply:
I think it's better if we stay consistent in this file and use the `Optional[X]` format here for now. We should, however, update occurrences of this format everywhere in the SDK. Although, that sounds out-of-scope for this PR to me.